config/src/main/java/com/alibaba/nacos/config/server/model/ConfigCacheGray.java

```diff
@@ -49,15 +51,6 @@ public GrayRule getGrayRule() {
         return grayRule;
     }
     
-    public ConfigCacheGray(String md5Gbk, String md5Utf8, long lastModifiedTs, String grayRule)
-            throws RuntimeException {
-        super(md5Gbk, md5Utf8, lastModifiedTs);
-        this.grayRule = GrayRuleManager.constructGrayRule(GrayRuleManager.deserializeConfigGrayPersistInfo(grayRule));
-        if (this.grayRule == null || !this.grayRule.isValid()) {
-            throw new RuntimeException("raw gray rule is invalid");
-        }
-    }
-    
     public String getGrayName() {
```

Comment:
miss code：

this.grayRule = GrayRuleManager.constructGrayRule(GrayRuleManager.deserializeConfigGrayPersistInfo(grayRule));
        if (this.grayRule == null || !this.grayRule.isValid()) {
            throw new RuntimeException("raw gray rule is invalid");
}